src/shadowsocks_config.spec.ts

```diff
@@ -221,6 +234,18 @@ describe('shadowsocks_config', () => {
       expect(LEGACY_BASE64_URI.stringify(config)).toEqual(
         'ss://YmYtY2ZiOnRlc3RAMTkyLjE2OC4xMDAuMTo4ODg4#Foo%20Bar');
     });
+
+    it('can serialize a legacy base64 URI with a non-latin password', () => {
+      const config = makeConfig({
+        host: '192.168.100.1',
+        port: '8888',
+        method: 'bf-cfb',
+        password: '小洞不补大洞吃苦',
+        tag: 'Foo Bar',
+      });
+      expect(LEGACY_BASE64_URI.stringify(config)).toEqual(
+        'ss://YmYtY2ZiOuWwj-a0nuS4jeihpeWkp-a0nuWQg-iLpkAxOTIuMTY4LjEwMC4xOjg4ODg#Foo%20Bar');
```

Comment:
This is violating the standard. It should be using base64, not base64url. See https://github.com/Jigsaw-Code/outline-client/issues/913#issuecomment-778518277